openslides/users/models.py

```diff
@@ -1,10 +1,7 @@
 # TODO: Check every app, that they do not import Group or User from here.
 
-from django.contrib.auth.models import (PermissionsMixin, AbstractBaseUser,
-                                        BaseUserManager)
-
-# TODO: Do not import the Group in here, but in core.models (if necessary)
```

Comment:
I added https://github.com/OpenSlides/OpenSlides/issues/1387

I don't think that we need a commend in the code, because the code is not our primary issue tracker.